neo4j/index.d.ts

```diff
@@ -0,0 +1,361 @@
+// Type definitions for node-neo4j 2.0.0
+// Project: https://github.com/thingdom/node-neo4j
+// Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../request/index.d.ts"/>
+
+declare module "neo4j" {
+    import { Request } from "request";
+
+    type CheckPasswordChangeNeededCallback = (error: any, changed: boolean) => void;
+    type CreateConstraintCallback = (error: any, constraint: any) => void;
+    type CypherCallback = (error: any, result: any) => void;
+    type DoneCallback = (error: any) => void;
+    type GetConstraintCallback = (error: any, constraints: any) => void;
+    type GetIndexesCallback = (error: any, indexes: any) => void;
+    type GetLabelCallback = (error: any, labels: any) => void;
+    type GetPropertyKeysCallback = (error: any, keys: any) => void;
+    type GetRelationshipTypesCallback = (error: any, types: any) => void;
+    type HasConstraintCallback = (error: any, exists: boolean) => void;
+    type HasIndexCallback = (error: any, exists: boolean) => void;
+    type HttpCallback = (error: any, body: any) => void;
+    type SaveCallback = (error: any, node: any) => void;
+
+    /**
+     * Authentication information.
+     * @interface
+     */
+    export interface Authentication {
+        /**
+         * Password.
```

Comment:
This doc comment isn't too useful -- below line already says that it's the password and it's a string.